pkg/controller/plan/migration.go

```diff
@@ -1109,9 +1082,14 @@ func (r *Migration) ensureGuestConversionPod(vm *plan.VMStatus) (err error) {
 	var vmCr VirtualMachine
 	found := false
 	if vmCr, found = r.vmMap[vm.ID]; !found {
-		msg := "VirtualMachine CR not found."
-		vm.AddError(msg)
-		return
+		vmCr.VirtualMachine, err = r.kubevirt.virtualMachine(vm)
+		if err != nil {
+			return
+		}
+		vmCr.DataVolumes, err = r.kubevirt.getDVs(vm)
```

Comment:
this seems to be problematic, i may need to switch everything to PVC here. i guess it worked for me because i don't have the latest CDI. in this stage the DV shouldn't be available.

Reply:
> this seems to be problematic, i may need to switch everything to PVC here.

makes sense, let's to it then